test/test_files/transaction/set_transaction.test

```diff
@@ -0,0 +1,71 @@
+-GROUP Set_Transaction
+-DATASET CSV tinysnb
+
+--
+
+
+#SingleTransactionReadWriteToFixedLengthStructuredNodePropertyNonNullTest
+-CASE NonNullTest
+-LOG Single Transaction Read Write To Fixed Length Structured Node Property NonNull Test
+-STATEMENT BEGIN WRITE TRANSACTION
+-STATEMENT MATCH (a:person) WHERE a.ID=0 RETURN a.age;
+---- 1
+35
+
+-STATEMENT MATCH (a:person) WHERE a.ID = 0 SET a.age = 70;
+---- ok
+
+-STATEMENT MATCH (a:person) WHERE a.ID=0 RETURN a.age;
+---- 1
+70
+
+#NullTest
+-CASE NullTest
+-LOG Single Transaction Read Write To Fixed Length Structured Node Property Null Test
+-STATEMENT BEGIN WRITE TRANSACTION
+-STATEMENT MATCH (a:person) WHERE a.ID=0 RETURN a.age;
+---- 1
+35
+
+-STATEMENT MATCH (a:person) WHERE a.ID = 0 SET a.age = null;
+---- ok
+
+-STATEMENT MATCH (a:person) WHERE a.ID=0 RETURN a.age;
+---- 1
+
+#Timeout
+-CASE TimeoutErrorTest
+-LOG Open Read Only Transaction Triggers Timeout Error For Write Transaction
+-CHECKPOINT_WAIT_TIMEOUT 10000
+-STATEMENT [conn1] BEGIN READ TRANSACTION;
```

Comment:
I wonder if it's better to create `conn1` first and then use it. Not it seems first looking for `conn1`, if it doesn't exist create one and use it.

Reply:
ok